web/src/showhidetoogle.js

```diff
@@ -0,0 +1,8 @@
+export function showhidetoogle(className) {
```

Comment:
export is only needed (and supported) in JS modules. In This case it has to be removed, so that the function can be found by the onclick handler.